axelor-account/src/main/resources/views/MoveLine.xml

```diff
@@ -175,7 +175,7 @@
       onChange="action-move-line-method-analytic-distribution-template-onchange"
       onSelect="action-move-line-method-analytic-distribution-template-onselect"
       readonlyIf="!account || !account.analyticDistributionAuthorized"
-      requiredIf="(!analyticMoveLineList || !analyticMoveLineList.length) &amp;&amp; account.analyticDistributionAuthorized &amp;&amp; account.analyticDistributionRequiredOnMoveLines &amp;&amp; (axis1AnalyticAccount == null &amp;&amp; axis2AnalyticAccount == null &amp;&amp; axis3AnalyticAccount == null &amp;&amp; axis4AnalyticAccount == null &amp;&amp; axis5AnalyticAccount == null)"
+      requiredIf="(!analyticMoveLineList || !analyticMoveLineList.length) &amp;&amp; (account.analyticDistributionAuthorized &amp;&amp; account.company.accountConfig.analyticDistributionTypeSelect != 3) &amp;&amp; account.analyticDistributionRequiredOnMoveLines &amp;&amp; (axis1AnalyticAccount == null &amp;&amp; axis2AnalyticAccount == null &amp;&amp; axis3AnalyticAccount == null &amp;&amp; axis4AnalyticAccount == null &amp;&amp; axis5AnalyticAccount == null)"
```

Comment:
No I said _"if the analytic distribution type is "Product/Family/Account", the analytic distribution template shouldn't be required on an account, event if the account has the boolean "analyticDistributionRequiredOnMoveLines" to true."_
Here you changed the MoveLine, not the Account. We don't want to change anything on the MoveLine.